### ISSUE TITLE
Automatically plot overlapping figures over the original ones

### ISSUE THREAD

**Issue by paulobruno** [natema/WorldDynamics.jl#92]
Inspired by the really nice [overlap figure](https://github.com/cvanwynsberghe/pyworld3/blob/main/img/result_standard_run.png) from @cvanwynsberghe in his [Python implementation of World3](https://github.com/cvanwynsberghe/pyworld3), we can create similar ones manually. It would be nice to automate the process for every figure.

**Comment by natema** [comment]
I think that automating this would be too complicated, and we would need to provide as artifacts all the figures of the books.  
I vote for closing this.

**Comment by paulobruno** [comment]
I agree that this is too complicated and far from being a priority. But it would be nice if someone at some point implement this.
So, I vote for leaving it open.
